### PR TITLE
Replace 'ancestry' with genetic ancestry group for callstats in VCF header

### PR DESCRIPTION
**Pull request by mike-w-wilson**

### CODE REVIEW

File: gnomad/utils/vcf.py [hunk 1]
```diff
@@ -706,11 +706,15 @@ def make_combo_header_text(
     if "subpop" in combo_dict or "pop" in combo_dict:
         if "subpop" in combo_dict:
             header_text = (
-                header_text + f" of {pop_names[combo_dict['subpop']]} ancestry"
+                header_text
+                + f" in the {pop_names[combo_dict['subpop']]} genetic ancestry group"
```
Comment (ch-kr) [inline gnomad/utils/vcf.py:710]:
```suggestion
                + f" in the {pop_names[combo_dict['subpop']]} genetic ancestry subgroup"
```
I'm not sure I feel strongly about this suggestion, though

File: gnomad/utils/vcf.py [hunk 2]
```diff
@@ -683,7 +683,7 @@ def make_combo_header_text(
     Programmatically generate text to populate the VCF header description for a given variant annotation with specific groupings and subset.
 
     For example, if preposition is "for", group_types is ["group", "pop", "sex"], and combo_fields is ["adj", "afr", "female"],
-    this function will return the string " for female samples of African-American/African ancestry".
+    this function will return the string " for female samples in the African-American/African genetic ancestry group".
```
Comment (ch-kr) [inline gnomad/utils/vcf.py:686]: nitpick, but we should also update gender terms to sex karyotypes in our docstrings

**Review by ch-kr — APPROVED** [review]
two minor suggestions but LGTM

File: gnomad/utils/vcf.py (hunk 1 above)
Comment (mike-w-wilson) [reply gnomad/utils/vcf.py:710]: Yeah I also thought about adding this but wasnt sure on the language, I think this makes sense though

File: gnomad/utils/vcf.py (hunk 2 above)
Comment (mike-w-wilson) [reply gnomad/utils/vcf.py:686]: I made https://app.zenhub.com/workspaces/gnomad-5f4d127ea61afc001d6be50b/issues/gh/broadinstitute/gnomad_production/1475 so this is tackled post 4.1

**Pull request merged**